docs/pages/includes/device-trust/prereqs.mdx

```diff
@@ -10,6 +10,8 @@
   - A user with permissions to use the /dev/tpmrm0 device (typically done by
     assigning the `tss` group to the user).
   - `tsh` v15.0.0 or newer. [Install tsh for Linux](../../installation.mdx#linux).
+  - WSL users should use the Windows binary instead.
```

Comment:
I would make this a separate paragraph within the previous bullet, rather than a new bullet. Otherwise, a reader may not connect this bullet to the previous one right a way.

Reply:
If I do something like this then the Windows download link is the last link that appears in the line, which I think makes it worse for the regular Linux user:

```
  - The `tsh` client. [Install tsh for Linux](../../installation.mdx#linux).
    WSL users should use the Windows binary instead.
    [Download the Windows tsh installer](../../installation.mdx#windows-tsh-and-tctl-clients-only).
```

WDYT? Did you have something different in mind?

Reply:
I was thinking:

```markdown
  - The `tsh` client. [Install tsh for Linux](../../installation.mdx#linux).

    WSL users should use the Windows binary instead.
    [Download the Windows tsh installer](../../installation.mdx#windows-tsh-and-tctl-clients-only).
```
This way, it's the same bullet, the Linux instructions are on their own line, and the Windows instructions are in a separate paragraph.

Reply:
Done!